src/utils/helpers.ts

```diff
@@ -0,0 +1,260 @@
+    let view = new DataView(arr);
+    if (bytes === 1) {
+      // byteOffset = 0;
+      view.setUint8(0, value);
+    } else if (bytes === 2) {
+      // byteOffset = 0; isLittleEndian = false
+      view.setUint16(0, value, false);
+    } else if (bytes === 4) {
+      // byteOffset = 0; isLittleEndian = false
+      view.setUint32(0, value, false);
+    }
+    return buffer.Buffer.from(arr);
+  },
+
+  /**
+   * Transform signed int to bytes (1, 2, or 4 bytes)
+   *
+   * @param {number} value Integer to be transformed to bytes
+   * @param {number} bytes How many bytes this number uses
+   *
+   * @return {Buffer} number in bytes
+   * @memberof Transaction
+   * @inner
+   */
+  signedIntToBytes(value: number, bytes: number): Buffer {
+    let arr = new ArrayBuffer(bytes);
+    let view = new DataView(arr);
+    if (bytes === 1) {
+      // byteOffset = 0
+      view.setInt8(0, value);
+    } else if (bytes === 2) {
+      // byteOffset = 0; isLittleEndian = false
+      view.setInt16(0, value, false);
+    } else if (bytes === 4) {
+      view.setInt32(0, value, false);
+    } else if (bytes === 8) {
+      // In case of 8 bytes I need to handle the int with a Long lib
+      let long = Long.fromNumber(value, false);
+      arr = long.toBytesBE();
+    }
+    return buffer.Buffer.from(arr);
+  },
+
+  /**
+   * Transform float to bytes
+   *
+   * @param {number} value Integer to be transformed to bytes
+   * @param {number} bytes How many bytes this number uses
+   *
+   * @return {Buffer} number in bytes
+   * @memberof Transaction
+   * @inner
+   */
+  floatToBytes(value: number, bytes: number): Buffer {
+    let arr = new ArrayBuffer(bytes);
+    let view = new DataView(arr);
+    if (bytes === 8) {
+      // byteOffset = 0; isLitteEndian = false
+      view.setFloat64(0, value, false);
+    }
+    return buffer.Buffer.from(arr);
+  },
+
+  /**
+   * Push data to the stack checking if need to add the OP_PUSHDATA1 opcode
+   * We push the length of data and the data
+   * In case the data has length > 75, we need to push the OP_PUSHDATA1 before the length
+   * We always push bytes
+   * 
+   * We update the array of Buffer sent as parameter, so we don't return a new one
+   * 
+   * @param {Array} stack Stack of bytes from the script
+   * @param {Buffer} data Data to be pushed to stack
+   *
+   * @memberof Transaction
+   * @inner
+   */
+  pushDataToStack(stack: Buffer[], data: Buffer) {
+    // In case data has length bigger than 75, we need to add a pushdata opcode
+    if (data.length > 75) {
+      stack.push(OP_PUSHDATA1);
+    }
+    stack.push(this.intToBytes(data.length, 1));
+    stack.push(data);
+  },
+  
+  /**
+   * Return the checksum of the bytes passed
+   * Checksum is calculated as the 4 first bytes of the double sha256
+   * 
+   * @param {Buffer} bytes Data from where the checksum is calculated
+   *
+   * @return {Buffer}
+   * @memberof Address
+   * @inner
+   */
+  getChecksum(bytes: Buffer): Buffer {
```

Comment:
I'd rename it to `getAddressChecksum(...)`.

Reply:
It was exactly this in the beginning but we don't use this checksum only for addresses. We use it to calculate the custom token configuration string as well. That's why I changed the name. What do you think?

Reply:
Ok. I usually don't like generic names because it is hard to figure out where it's been used.